extensions/sonarqube/overview.md

```diff
@@ -1,50 +1,59 @@
-**[SonarQube][sq]** is the leading product for Continuous Code Quality. It supports all major programming languages, including C#, VB .Net, JavaScript, TypeScript, C/C++ and many more. Integrate it in your on-premise TFS installation, and continuously track down bugs and vulnerabilities in your codebase.
+**[SonarQube™][sq]** is the leading tool for continuously inspecting the Code Quality and Security™ of your codebases, all while empowering development teams. Covering 25+ programming languages including C#, VB.Net, JavaScript, TypeScript and C++; SonarQube easily pairs up with your Azure DevOps environment and tracks down bugs, vulnerabilities and code smells. With over 170,000 deployments helping small development teams as well as global organizations, SonarQube provides the means for all teams and companies around the world to own and impact their Code Quality and Security.
```

Comment:
"Covering 27 languages"?
Also: "tracks down Bugs, Security Vulnerabilities, and Code Smells"

Reply:
Are you suggesting an Oxford comma or the insertion of 'Security' or both?

Reply:
I can take or leave the oxford comma. What I'm suggesting is our proper nouns, i.e. init caps _and_ the insertion of 'Security'